Update docs (#121)
threefoldtech/rmb-rs: README.md

[![Rust](https://github.com/threefoldtech/rmb-rs/actions/workflows/rust.yaml/badge.svg)](https://github.com/threefoldtech/rmb-rs/actions/workflows/rust.yaml)

# Reliable Message Bus
Reliable message bus is a secure communication panel that allows `bots` to communicate together in a `chat` like way. It makes it very easy to host a service or a set of functions to be used by anyone, even if your service is running behind NAT.

Out of the box RMB provides the following:

> - Grantee authenticity of the messages. You are always sure that the received message is from whoever is pretending to be

Reviewer: ```suggestion
- Guarantee authenticity of the messages. You are always sure that the received message is from whoever is pretending to be
```

- End to End encryption

> - Support for 3rd party hosted relays. Anyone can host a relay and people can use it safely since there is noway messages can be inspected while using e2e. That's similar to `home` servers by `matrix`

Reviewer: ```suggestion
- Support for 3rd party hosted relays. Anyone can host a relay and people can use it safely since there is no way messages can be inspected while using e2e. That's similar to `home` servers by `matrix`
```

## Why
RMB is developed by ThreefoldTech to create a global network of nodes that are available to host capacity. Each node will act like a single bot where you can ask to host your capacity. This enforced a unique set of requirements:
- Communication needed to be reliable
  - Minimize and completely eliminate message loss
  - Reduce downtime
- Node need to authenticate and authorize calls

>   - Grantee identity of the other peer so only owners of data can see it

Reviewer: ```suggestion
  - Guarantee identity of the other peer so only owners of data can see it
```
